.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,9 @@ ran. If your test fixes a bug reported in an issue, don't forget to include the
 issue number. If your PR is still a work in progress, that's totally fine – just
 include a note to let us know. -->
 
+### Corresponding documentation PR
+<!--- Add the link to the corresponding documentation PR here, if applicable. -->
+
 ### Types of change
```

Comment:
These edits shouldn't be showing up in the diff of this PR. Can you merge in the current `develop` so they hopefully go away? It'll allow us to focus review on the actual differences this PR introduces.